tests/TestHelpers/EmailHelper.php

```diff
@@ -0,0 +1,174 @@
+<?php declare(strict_types=1);
+/**
+ * ownCloud
+ *
+ * @author Prajwol Amatya <prajwol@jankaritech.com>
+ * @copyright Copyright (c) 2023 Prajwol Amatya prajwol@jankaritech.com
+ */
+
+namespace TestHelpers;
+
+use Exception;
+use GuzzleHttp\Exception\GuzzleException;
+use Psr\Http\Message\ResponseInterface;
+
+/**
+ * A helper class for managing emails
+ */
+class EmailHelper {
+	/**
+	 * @param string $emailAddress
+	 *
+	 * @return string
+	 */
+	public static function getMailBoxFromEmail(string $emailAddress):string {
+		return explode("@", $emailAddress)[0];
+	}
+
+	/**
+	 * Returns the host and port where Email messages can be read and deleted
+	 * by the test runner.
+	 *
+	 * @return string
+	 */
+	public static function getLocalEmailUrl():string {
+		$localEmailHost = self::getLocalEmailHost();
+		$emailPort = \getenv('EMAIL_PORT');
+		if ($emailPort === false) {
+			$emailPort = "9000";
+		}
+		return "http://$localEmailHost:$emailPort";
+	}
+
+	/**
+	 * Returns the host name or address of the Email server as seen from the
+	 * point of view of the system-under-test.
+	 *
+	 * @return string
+	 */
+	public static function getEmailHost():string {
+		$emailHost = \getenv('EMAIL_HOST');
+		if ($emailHost === false) {
+			$emailHost = "127.0.0.1";
+		}
+		return $emailHost;
+	}
+
+	/**
+	 * Returns the host name or address of the Email server as seen from the
+	 * point of view of the test runner.
+	 *
+	 * @return string
+	 */
+	public static function getLocalEmailHost():string {
+		$localEmailHost = \getenv('LOCAL_EMAIL_HOST');
+		if ($localEmailHost === false) {
+			$localEmailHost = self::getEmailHost();
+		}
+		return $localEmailHost;
+	}
+
+	/**
+	 * Returns general response information about the provided mailbox
+	 * A mailbox is created automatically in InBucket for every unique email sender|receiver
+	 *
+	 * @param string $mailBox
+	 * @param string|null $xRequestId
+	 *
+	 * @return mixed JSON encoded contents
+	 * @throws GuzzleException
+	 */
+	public static function getMailBoxInformation(string $mailBox, ?string $xRequestId = null) {
```

Comment:
```suggestion
	public static function getMailBoxInformation(string $mailBox, ?string $xRequestId = null):mixed {
```

Reply:
It throw error while using mixed so used array and object for getMailBoxInformation and getBodyOfAnEmailById functions respectively.